public/static/app/perf/TrendSamples.js

```diff
@@ -103,32 +102,36 @@ mciModule.factory('TrendSamples', function() {
       return this._tasks;
     }
 
-    this.tasksByCommitOrderByTestName = function(testName){
-       if(!(testName in this._tasksByName)){
-          this._tasksByName[testName] = _.sortBy(_.uniq(this.seriesByName[testName], function(x){return x.task_id}), "order")
-       }
-       return this._tasksByName[testName]
+    this.tasksByCommitOrderByTestName = function (testName) {
+      if (!(testName in this._tasksByName)) {
+        this._tasksByName[testName] = _.sortBy(_.uniq(this.seriesByName[testName], function (x) {
```

Comment:
```
_.chain(this.seriesByName[testName]).sortBy("order").uniq(true, x =>{return x.task_id}).value()
```

Reply:
Sorry, this got changed by my formatter. I'd prefer to not change this logic unless there's a specific reason to